server/webapp/WEB-INF/rails.new/spec/javascripts/websocket_wrapper_spec.js

```diff
@@ -0,0 +1,318 @@
+/*
+ * Copyright 2017 ThoughtWorks, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+describe("Websocket Wrapper", function () {
+  var MockedWebsocket, oftenWrapper;
+
+  beforeEach(function () {
+    oftenWrapper = {
+      often: jasmine.createSpy(),
+      start: jasmine.createSpy(),
+      wait:  jasmine.createSpy(),
+      done:  jasmine.createSpy()
+    };
+
+    often = oftenWrapper.often;
```

Comment:
This will overwrite the global, and leave it as is. Might have an impact on other tests. Might make sense to rollback these vars to their original state?